src/wcs2_0/src/test/java/org/geoserver/wcs2_0/kvp/GetCapabilitiesTest.java

```diff
@@ -125,6 +125,35 @@ public void testAcceptVersions() throws Exception {
                 "0", "count(//ows:ExceptionReport//ows:Exception[@locator='wCS'])", dom);
     }
 
+    @Test
+    public void testInternationalContent() throws Exception {
+        GeoServerInfo global = getGeoServer().getGlobal();
+        global.getSettings().setProxyBaseUrl("src/test/resources/geoserver");
+        getGeoServer().save(global);
+        Catalog catalog = getCatalog();
+        CoverageInfo ci = catalog.getCoverageByName(getLayerId(TASMANIA_DEM));
+        ci.setTitle("My Title");
+        ci.setAbstract("My Abstract");
+        ci.getKeywords().add(0, new Keyword("my_keyword"));
+        MetadataLinkInfo mdl1 = catalog.getFactory().createMetadataLink();
+        mdl1.setContent("http://www.geoserver.org/tasmania/dem.xml");
+        mdl1.setAbout("http://www.geoserver.org");
+        ci.getMetadataLinks().add(mdl1);
+        MetadataLinkInfo mdl2 = catalog.getFactory().createMetadataLink();
+        mdl2.setContent("/metadata?key=value");
+        mdl2.setAbout("http://www.geoserver.org");
+        ci.getMetadataLinks().add(mdl2);
+        catalog.save(ci);
+        Document dom =
```

Comment:
The parameter that should be used to request a specific language in getCapabilities is the AcceptLanguages one.

Language request parameter is meant to be used only if the Inspire plug-in is at stake.

Reply:
Changed to Accept Languages.